templates/page.html

```diff
@@ -185,19 +185,20 @@ <h3>Select a REANA cluster...</h3>
         <h3>...or install your own</h3>
         <figure class="figure">
           <div class="command-line">
+            <!-- FIXME: Should we recommend kubectl 1.21 already? -->
```

Comment:
I'm going to need some input here.

Reply:
We can still keep this, since 1.16+ versions should work nicely. Or perhaps increase to 1.19+ which is what we battle-tested in production.  If we show only 1.21+, people might think the REANA doesn't run on earlier versions. So I'd propose to mention kubectl 1.19+, kind 0.9+, helm 3.0+.